<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test1</class>
 <widget class="QWidget" name="Test1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="but_init">
       <property name="text">
        <string>创建excel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="but_open">
       <property name="text">
        <string>打开excel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="but_write">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>写入数据</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="but_recv">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>读取数据</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="but_saveAs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>另存为</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="but_close">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>关闭excel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>166</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
